otter/json_schema/group_schemas.py

```diff
@@ -321,6 +322,21 @@
 }
 
 
+def validate_launch_config_servicenet(lc):
+    """
+    Validate that if CLBs are provided, ServiceNet is also provided.
+    """
+    clb = any([lb.get('type', 'CloudLoadBalancer') == 'CloudLoadBalancer'
+               for lb in get_in(('args', 'loadBalancers'), lc, default=())])
+    servicenet = get_in(('args', 'server', 'networks'), lc, default=None)
```

Comment:
Yes, sorry, the naming should be `networks` - the next line lets it go if it's None.